sakura_core/mem/CPoolResource.h

```diff
@@ -65,6 +65,8 @@ class CPoolResource : public std::pmr::memory_resource
  	void* do_allocate([[maybe_unused]] std::size_t bytes,
  					  [[maybe_unused]] std::size_t alignment) override
 	{
+		std::lock_guard<std::mutex> lock(mtx);
```

Comment:
(sonarcloud指摘)　対応必須ではありませんが、メンテナンス性は上がりそう。

```suggestion
		std::lock_guard lock(mtx);
```